mycroft/util/parse.py

```diff
@@ -104,8 +107,8 @@ def extract_numbers(text, short_scale=True, ordinals=False, lang=None):
     Returns:
         list: list of extracted numbers as floats, or empty list if none found
     """
-    return lingua_franca.parse.extract_numbers(text, short_scale,
-                                               ordinals, lang)
+    return lingua_franca.parse.extract_numbers(text, short_scale, ordinals,
+                                               lang or get_default_lang())
```

Comment:
LF should be handling `lang=None` with a DeprecationWarning. Should Mycroft wrap and fix this? It'll make for cleaner logs at the consumer side, but Skill devs won't catch the warnings.

Reply:
Yeah - I did it to quiet the 797 warnings you get from the unit tests :scream: 

But it's not necessary and we should be able to drop most (if not all) of the wrappers over time.

Reply:
Ironically, that DeprecationWarning is ahead of a custom exception that could be ignored XD